tests/unit/LedgerConsentContinuationOut.spec.ts

```diff
@@ -2,16 +2,15 @@ import Vue from 'vue'
 import Vuetify from 'vuetify'
 import { mount } from '@vue/test-utils'
 import ConsentContinuationOut from '@/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue'
-import { getVuexStore } from '@/store'
+import { createPinia, setActivePinia } from 'pinia'
 
 Vue.use(Vuetify)
 const vuetify = new Vuetify({})
-const store = getVuexStore() as any // remove typings for unit tests
+setActivePinia(createPinia())
```

Comment:
I think this component doesn't need the store -- this can be removed.

Reply:
I was wondering why I had this here. I tried running the test without the component. I was getting the no active Pinia error that I mentioned in main.ts. So, I had to add it in order for the tests to run.